crm_win_reason/wizard/crm_lead_won.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class CrmLeadReason(models.TransientModel):
+    _name = "crm.lead.won"
+    _description = "Get Won Reason"
+
+    win_reason_id = fields.Many2one(
+        "crm.lost.reason", "Won Reason", domain="[('reason_type','=','won')]"
```

Comment:
i don't like the fact that model has still 'lost' in its name even it is used for won reason but ATM functionality is there, can be changed later.